storage/statedb/derive_sha.go

```diff
@@ -31,9 +31,25 @@ import (
 type DeriveShaOrig struct{}
 
 func (d DeriveShaOrig) DeriveSha(list types.DerivableList) common.Hash {
+	trie := NewStackTrie(nil)
+	trie.Reset()
```

Comment:
@hqjang-pepper Is this necessary right after a new stackTrie is created?

Reply:
Yes Reset() is an initializing function of `stackTrie`, so no problem at all.